prime-router/src/main/kotlin/azure/HistoryFunctions.kt

```diff
@@ -404,58 +403,61 @@ open class BaseHistoryFunction : Logging {
      * returns null if not authorized, otherwise returns a set of claims.
      */
     fun checkAuthenticated(request: HttpRequestMessage<String?>, context: ExecutionContext): AuthClaims? {
-        var userName = ""
-        // orgs in the settings table of the database have a format of "zz-phd",
-        // while the auth service claims has a format of "DHzz_phd"
-        // claimsOrgName will have the format of "DHzz_phd"
-        val claimsOrgName = request.headers["organization"] ?: ""
-
-        // orgName will have the format of "zz-phd" and is used to look up in the settings table of the database
-        var orgName = getOrgNameFromHeader(claimsOrgName)
-
-        var jwtToken = request.headers["authorization"] ?: ""
+        val userName: String? /* Format: email */
+        val requestOrgName = request.headers["organization"] ?: "" /* Format: xx-phd */
+        val oktaOrganizations: List<String?>
+        val accessOrgName: String? /* Format: xx-phd */
+        var jwtToken = request.headers["authorization"] ?: "" /* Format: Bearer ... */
+
```

Comment:
From the above code:

1. Can requestOrgName be a `var`? (maybe turn on "Problems" in IntelliJ)

2. Some items get defaults (e.g. "") and others get undefined/null. It seems inconsistent within the function.

Was there a reason `userName`, `oktaOrganizations` and `accessOrgName` don't just get defaulted to `""`, `[]` and `""`? (I suspect it's just left over from the original code.)

From code inspection if the try{}catch throws, then userName may not be changed from undefined/null and then further down in the code there is: 
```kotlin
  if (userName.isBlank() || accessOrgName.isNullOrBlank()) return null
```

It would be nice to pick a standard approach. `isNullOrBlank()` seems better.

Then there's this bit of code:
```kotlin
accessOrgName = when {
                oktaOrganizations.contains("DHPrimeAdmins") ||
                    oktaOrganizations.contains(toOktaOrgName(requestOrgName)) -> requestOrgName
                else -> null
```
Which could set `accessOrgName` to `null` (not `""`)

Reply:
lol... I re-reviewed and saw the same things. That's what happens when I'm doing too many things at once.

Reply:
I actually did go back to standardize things to nullable strings and `isNullOrBlank()`. I decided the perfectionist in me would win this time. 😂